docs/getting-started/linux-installer.md

```diff
@@ -183,14 +185,21 @@ To upgrade the Collector, run the following commands on your system (requires
 
 ### Fluentd Configuration
 
-> If log collection is not required, run the installer script with the
-> `--without-fluentd` option to skip installation of Fluentd and the
-> plugins/dependencies listed below.
+If log collection with Fluentd is required and your platform is [supported](
+#supported-platforms), run the installer script with the `--with-fluentd`
+option to install/configure Fluentd and the plugins/dependencies listed below.
+For example:
+
+```sh
+curl -sSL https://dl.signalfx.com/splunk-otel-collector.sh > /tmp/splunk-otel-collector.sh && \
+sudo sh /tmp/splunk-otel-collector.sh --realm SPLUNK_REALM --with-fluentd -- SPLUNK_ACCESS_TOKEN
+```
 
-By default, the Fluentd service will be installed and configured to forward log
-events with the `@SPLUNK` label to the Collector (see below for how to add
-custom Fluentd log sources), and the Collector will send these events to the
-HEC ingest endpoint determined by the `--realm SPLUNK_REALM` option, e.g.
+The Fluentd deb/rpm package (`td-agent`) will be installed, and the `td-agent`
+service will be configured to forward log events with the `@SPLUNK` label to
+the Collector (see below for how to add custom Fluentd log sources). The
+Collector will then send these events to the HEC ingest endpoint determined by
+the `--realm SPLUNK_REALM` option, i.e.
```

Comment:
Does it make sense to still default the HEC endpoint to the realm?

Reply:
I believe this was handled in https://github.com/signalfx/splunk-otel-collector/pull/3330.